docs/ATLAS.md

```diff
@@ -0,0 +1,23 @@
+### Setup Atlas using terraform ###
+
+Follow these steps to setup a M2 cluster in Atlas to be used by the uDeploy portal.
+
+1. Register an API key in Atlas.  `Atlas -> Project -> Access management -> API Keys`
+    
+    - Click `Manage -> Create API Key`
+    - Enter a short description for this key
+    - Give the key `Project Owner` permission (This permission will be reduced at the end of these steps)
```

Comment:
I would remove `(This permission will be reduced at the end of these steps)` since this is the users option and not important for udeploy setup.